api/v1alpha1/suite_test.go

```diff
@@ -46,7 +47,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func(done Done) {
-	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+	logf.SetLogger(log)
```

Comment:
`logr` is an interface, do the logs still get output here during unit-tests?

Reply:
Yes, we did.